src/coreclr/gc/env/gcenv.object.h

```diff
@@ -11,7 +11,7 @@ extern bool g_oldMethodTableFlags;
 // ARM requires that 64-bit primitive types are aligned at 64-bit boundaries for interlocked-like operations.
 // Additionally the platform ABI requires these types and composite type containing them to be similarly
 // aligned when passed as arguments.
-#ifdef TARGET_ARM
+#if defined(TARGET_ARM) || defined(TARGET_BROWSER)
```

Comment:
It would be better to use arch-specific ifdefs (`TARGET_WASM`) than OS-specific ifdefs for things like this that will be common to all 32 bit WASM OSes.

Reply:
Yeah, I changed it to TARGET_WASM as the alignment should be the same.